doc/content/getting_started/installation.md

```diff
@@ -117,7 +117,7 @@ repository, use the following commands.
 ```bash
 cd ~/projects/TMAP8
 git fetch upstream
-git rebase upstream/main
+git rebase upstream/devel
```

Comment:
We put `main` in the original version of these instructions because we wanted general users to use the most stable branch instead of devel, which should be for developers. Perhaps we need to be clearer here about which update is recommended for each group?

Reply:
I just updated these instructions to provide both, with clarification on who it applies to. Let me know if that looks good to you.

Reply:
Made some follow-up comments, but I think we're good after this round. Thanks!